frontend/packages/core/src/popper.tsx

```diff
@@ -89,7 +89,7 @@ export interface PopperProps
   extends Pick<ClickAwayListenerProps, "onClickAway">,
     Pick<MuiPopperProps, "placement"> {
   open: boolean;
-  anchorRef: React.MutableRefObject<HTMLElement>;
+  anchorRef: React.MutableRefObject<HTMLElement> | null;
```

Comment:
```suggestion
  anchorRef?: React.MutableRefObject<HTMLElement>;
```

Reply:
https://stackoverflow.com/questions/57243477/can-an-optional-parameter-be-null-in-typescript

Item 5 shows that this approach of using optional to allow for null does not work (it does for `undefined` but not `null`)

Reply:
ah true, can we have both then? i think both values make sense i.e. `anchorRef?: React.MutableRefObject<HTMLElement> | null`

Reply:
we can